src/main/java/ch/njol/skript/expressions/ExprDurability.java

```diff
@@ -80,25 +86,27 @@ public Class<?>[] acceptChange(final ChangeMode mode) {
 		}
 		return null;
 	}
-	
-	@SuppressWarnings("null")
+
 	@Override
-	public void change(final Event e, final @Nullable Object[] delta, final ChangeMode mode) {
+	@SuppressWarnings("null")
+	public void change(Event event, @Nullable Object[] delta, ChangeMode mode) {
 		int a = delta == null ? 0 : ((Number) delta[0]).intValue();
```

Comment:
We need a better variable name than `a`. The letter A has no reference to a Number or anything about this expression. Looking at just `a` I have no clue what that could be.

Reply:
I just left it how it was, what should I change it to